samples/client/petstore-security-test/scala/build.gradle

```diff
@@ -104,6 +104,12 @@ ext {
     jackson_version = "2.4.2"
     junit_version = "4.8.1"
     scala_test_version = "2.2.4"
+    swagger_async_httpclient_version = "0.3.5"
```

Comment:
It looks like this wasn't added by this PR, but I haven't seen it elsewhere yet.

swagger-async-httpclient hasn't been updated in 3 years. Is there any reason we're not using an async http client that is current?

Reply:
Hmm, not sure where this came from. Just a side effect of running the petstore sample scripts.

Reply:
The Petstore security test samples (Scala) were not updated before.

Reply:
Sorry, I should have explicitly mentioned that my second question was a general aside. It makes sense that the script added it, and the sample just hadn't been generated in a while.